options/m_option.h

```diff
@@ -394,11 +394,14 @@ struct m_option {
 // Do not add as property.
 #define M_OPT_NOPROP            (1 << 6)
 
+// This options prints possible values when called as --option=help
+#define M_OPT_HELP              (1 << 7)
```

Comment:
Why not just add this as `(1 << 9)` then the other flags don't need to be changed.

Reply:
@eintw1ck to keep `M_OPT_` defines separated from `UPDATE_` defines, I suppose. The list of defines looks nicer this way.

Reply:
Yeah, I wanted to keep them in order and in their respective groups. In this history these have been reordered and old ones replaced sometimes, so I didn't think it'd be a problem to bump them. The values are only used internally.